api/v3/common_types.go

```diff
@@ -107,6 +107,9 @@ type CommonSplunkSpec struct {
 	// LicenseMasterRef refers to a Splunk Enterprise license manager managed by the operator within Kubernetes
 	LicenseMasterRef corev1.ObjectReference `json:"licenseMasterRef"`
 
+	// LicenseManagerRef refers to a Splunk Enterprise license manager managed by the operator within Kubernetes
+	LicenseManagerRef corev1.ObjectReference `json:"licenseManagerRef"`
```

Comment:
change this to optional field so one of the field is selected


`//+optoinal
LicenseMasterRef corev1.ObjectReference `json:"licenseMasterRef,omitempty"`

//+optional
LicenseManagerRef corev1.ObjectReference `json:"licenseManagerRef,omitempty"``

Reply:
Marked both *Master and *Manager as optional in the latest commit.